tests/cpu/Context_tests.cpp

```diff
@@ -1,7 +1,6 @@
 // SPDX-License-Identifier: BSD-3-Clause
 // Copyright Contributors to the OpenColorIO Project.
 
-
```

Comment:
The changes are on line 121 and 145 to remove references to the now deleted `GPUHelpers.h` file.

My editor is setup to run clang-format automatically, but I can see how this could be annoying.
I'll revert the whitespace changes.